bin/main.rb

```diff
@@ -1,2 +1,110 @@
-#!/usr/bin/env ruby
-puts 'Hello world!'
+# rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+
+# !/usr/bin/env ruby
```

Comment:
- [ ] For this to work, the top line should be the one to make it executable. I think your Autosave is giving you troubles. It should be like this:

![image](https://user-images.githubusercontent.com/61479648/93911470-cc5cf100-fcd8-11ea-87b6-c3eba4efb997.png)

- [ ] You are disabling the Perceived Complexity. That is a good rule to make your methods short and precise. You should not do this. Try to divide your methods more. In this case your program will be easier to maintain.